entities/communication/voiceCallSetting.json

```diff
@@ -75,8 +75,14 @@
         "staff_weekly_availability_uid": "0c4ac9717ab26f56",
         "call_timeout_in_sec": 20,
         "voice_scripts": {
-            "greeting_message": "Hi, The call will be forwarded soon, please wait.",
-            "missed_call_message": "Sorry, please try again later."
+            "greeting_message": {
+                "message": "Hi, The call will be forwarded soon, please wait.",
+                "enabled": true
+            },
+            "missed_call_message": {
+                "message": "Sorry, please try again later.",
+                "enabled": false
+            }
         },
```

Comment:
You need to update the description of the voice_scripts property to align with this change.

This is what I got from ChatGPT:
The voice_scripts property within the voiceCallSettings entity allows for the configuration of various voice messages that are played to callers during different stages of a voice call. Each key-value pair within voice_scripts represents a specific voice message, identified by a unique name.

Structure:
key: String - The name of the voice message configuration (e.g., greeting_message, missed_call_message). This key can be any descriptive name suitable for the specific message.

value: Object - Contains the following properties:

message: String - The actual voice message that will be played to the caller.
enabled: Boolean - A flag indicating whether this specific voice message is enabled or disabled.

Reply:
Ok sure, will update accordingly, thanks :)

Reply:
@ramsterita Done. thanks